src/phoenix/server/api/routers/v1/experiment_evaluations.py

```diff
@@ -0,0 +1,60 @@
+from datetime import datetime
+
+from starlette.requests import Request
+from starlette.responses import JSONResponse, Response
+from starlette.status import HTTP_404_NOT_FOUND
+from strawberry.relay import GlobalID
+
+from phoenix.db import models
+from phoenix.server.api.types.node import from_global_id_with_expected_type
+
+
+async def create_experiment_evaluation(request: Request) -> Response:
+    experiment_run_globalid = GlobalID.from_id(request.path_params["run_id"])
+    try:
+        experiment_run_id = from_global_id_with_expected_type(
+            experiment_run_globalid, "ExperimentRun"
+        )
+    except ValueError:
+        return Response(
+            content=f"ExperimentRun with ID {experiment_run_globalid} does not exist",
+            status_code=HTTP_404_NOT_FOUND,
+        )
+
+    payload = await request.json()
+    name = payload.get("name")
+    label = payload.get("label")
+    score = payload.get("score")
+    explanation = payload.get("explanation")
+    error = payload.get("error")
+    metadata = payload.get("metadata", {})
+    start_time = payload.get("start_time")
+    end_time = payload.get("end_time")
+    async with request.app.state.db() as session:
+        experiment_evaluation = models.ExperimentEvaluation(
+            experiment_run_id=int(experiment_run_id),
+            name=name,
+            label=label,
+            score=score,
+            explanation=explanation,
+            error=error,
+            metadata_=metadata,
+            start_time=datetime.fromisoformat(start_time),
+            end_time=datetime.fromisoformat(end_time),
+        )
+        session.add(experiment_evaluation)
+        await session.flush()
```

Comment:
Any reason not to use the `insert` API? I think it does not require an explicit flush.